pkg/config/serving/server.go

```diff
@@ -58,7 +58,10 @@ func ToServerConfig(ctx context.Context, servingInfo configv1.HTTPServingInfo, a
 	}
 
 	if !authorizationConfig.Disabled {
-		authorizationOptions := genericapiserveroptions.NewDelegatingAuthorizationOptions()
+		authorizationOptions := genericapiserveroptions.NewDelegatingAuthorizationOptions().
+			WithAlwaysAllowPaths("/healthz", "/healthz/", "/readyz"). // this allows the kubelet to always get health and readiness without causing an access check
```

Comment:
> healthz and readyz have to accessible to unauthenticated so the kubelet can health check. Because of this, there is no reason to run an authz check on them.

how does it work today, then?

Reply:
> how does it work today, then?

it does a subject access review  (remote call)